src/Nethermind/Nethermind.Blockchain.Test/Bloom/BloomStorageTests.cs

```diff
@@ -123,6 +123,27 @@ public void Returns_proper_blooms_after_store(long from, long to, bool isMatch,
             ranges.Should().BeEquivalentTo(expectedBlocks);
             bloomsChecked.Should().Be(expectedBloomsChecked);
         }
+        
+        [Test]
+        public void Can_find_bloom_offseted()
+        {
+            var storage = CreateBloomStorage(new BloomConfig() {IndexLevelBucketSizes = new []{4}});
```

Comment:
can you add tests for multilevel cases please

Reply:
something like 60, 12, 3

Reply:
Added more complex test cases